literate_notebooks/src-ES/01_constructors.jl

```diff
@@ -0,0 +1,145 @@
+# # Introducción a DataFrames
+# **[Bogumił Kamiński](http://bogumilkaminski.pl/about/), May 23, 2018**
+# (Traducción por Miguel Raz, Abril 16 2021)
+# 
+# Empecemos cargando el paquete de `DataFrames`.
+
+using DataFrames
+
+# ## Constructores y conversiones
+
+#-
+
+# ### Constructores
+# 
+# En esta secciónn, verás distintas maneras de crear un `DataFrame` usando el constructor `DataFrame()`.
+# 
+# Primero, creemos un DataFrame vacío.
+
+DataFrame() #  DataFrame vacío
+
+# O podemos llamar al constructor usando keyword arguments para agregar columnas al `DataFrame`.
+
+DataFrame(A=1:3, B=rand(3), C=randstring.([3,3,3]))
+
+# Podemos creat el `DataFrame` de un diccionario, en cuyo caso las llaves del diccionario estarán ordenadas para crear las columnas del `DataFrame`.
+
+x = Dict("A" => [1,2], "B" => [true, false], "C" => ['a', 'b'])
+DataFrame(x)
+
+# En vez de explícitamente crear el diccionario primero, como hicimos arriba, podríamos pasar argumentos de `DataFrame` con la sintaxis de pares llave-valor de un diccionario.
+# 
+# Notar que en este caso, usamos símbolos para denotar los nombres de las columnas y los argumentos no están ordenados. Por ejemplo, `:A`, el símbolo, produce `A`, el nombre de la primera columna:
+
+DataFrame(:A => [1,2], :B => [true, false], :C => ['a', 'b'])
+
+# Aquí creamos un `DataFrame` de un vector de vectores, donde cada vector se convierte en una columna.
+
+DataFrame([rand(3) for i in 1:3])
+
+#  Por ahora podemos construir un `DataFrame` de un `Vector` de átomos, creando un `DataFrame` con una sola hilera. En versiones futuras de DataFrames.jl, esto arrojará un error.
```

Comment:
Should we delete this? Since it will be a 1.0 compatible anyway so there's no need to talk about deprecations.

Reply:
Yeah there's a couple of pre DataFrames 1.0 kinks that can be cleaned up. I'm waiting to run the notebooks and test that everything works to filter many of these out. I still don't know much about DataFrames so I don't know what is/isn't design yet.